widgets/controls.pas

```diff
@@ -1873,8 +1873,8 @@ procedure TControl.AlignControls;
         VControl.BeginUpdate;
         try
           VSpacing := VControl.BorderSpacing;
-          if not (akLeft in VControl.Anchors) then
-            VControl.Left := VRight - VControl.Width - VSpacing.Right - VSpacing.Around;
+//          if not (akLeft in VControl.Anchors) then
+          VControl.Left := VRight - VControl.Width - VSpacing.Right - VSpacing.Around;
           VControl.Top := VTop + VSpacing.Top + VSpacing.Around;
```

Comment:
I'll need to check whether this breaks anything with anchors (cause there was a reason why I added this), but if it doesn't break anything then you can just remove that line instead of commenting it).